### PR TITLE
PA-35 Added SCA Service for IDE extension work

### PR DESCRIPTION
**Pull request by SOOS-JAlvarez**
### Changes:
  - Description of the change
  - Updated package.json version

**Ticket:** https://soos.atlassian.net/browse/PA-35

### CODE REVIEW

File: src/services/AnalysisService.ts [hunk 1]
```diff
@@ -453,6 +460,135 @@ class AnalysisService {
     return { filePaths: filesToUpload, hasMoreThanMaximumFiles };
   }
 
+  async searchManifestFiles({
```
Comment (SOOS-JAlvarez) [inline src/services/AnalysisService.ts:463]: TODO - See if we can consolidate this method with `findAnalysisFiles`

File: src/services/AnalysisService.ts [hunk 2]
```diff
@@ -33,6 +34,12 @@ interface IGenerateFormattedOutputParams {
   workingDirectory: string;
 }
 
+interface IManifestFile {
+  packageManager: string;
```
Comment (SOOS-GSteen) [inline src/services/AnalysisService.ts:38]: enum for package manager

File: src/services/AnalysisService.ts [hunk 3]
```diff
@@ -453,6 +463,137 @@ class AnalysisService {
     return { filePaths: filesToUpload, hasMoreThanMaximumFiles };
   }
 
+  async searchManifestFiles({
```
Comment (SOOS-GSteen) [inline src/services/AnalysisService.ts:466]:
```suggestion
  async findManifestFiles({
```

File: src/services/AnalysisService.ts (hunk 1 above)
Comment (SOOS-JAlvarez) [reply src/services/AnalysisService.ts:466]: will do later on

**Review by SOOS-GSteen — APPROVED** [review]

**Pull request merged**